aggregator_server/aggregator_server.cpp

```diff
@@ -108,7 +108,7 @@ class aggregator_server final : public zecale_proto::Aggregator::Service
     grpc::Status RegisterApplication(
         grpc::ServerContext * /*context*/,
         const zecale_proto::ApplicationRegistration *registration,
-        proto::Empty *response) override
```

Comment:
What about `proto::Empty * /*response*/)`? (like I did for the context above) I think this makes clang-tidy happier

Reply:
Switched to `/*response*/`

Reply:
`proto::Empty * /*request*/) override` -> `proto::Empty * /*response*/) override`